### PR TITLE
TA-224: Provider tests for ExpertUi ManageOrg

### PR DESCRIPTION
**Pull request by ruban72**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/OrganisationIdArgumentResolver.java [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
+import uk.gov.hmcts.reform.professionalapi.repository.ProfessionalUserRepository;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.ProfessionalApiConstants.ERROR_MESSAGE_403_FORBIDDEN;
+
+
+@Component
+@Slf4j
+public class OrganisationIdArgumentResolver implements HandlerMethodArgumentResolver {
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/OrganisationIdArgumentResolver.java:28]: again it shows all file has been changed and looks like only some stuff changed

**Review by abhijitdiwan — APPROVED** [review]

**Review by abhijitdiwan — DISMISSED** [review]

**Review by shreedhar-hmcts — COMMENTED** [review]
I do not see professionalInternalController provider test cases ? 
could you please remove commented code?

File: src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/OrganisationIdArgumentResolver.java (hunk 1 above)
Comment (ruban72) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/resolver/OrganisationIdArgumentResolver.java]: This has been removed from change list now

**Comment by nitishsingla1234** [comment]
Hi @ruban72 Will we be able to find the Pact test report in Jenkins build? I tried looking at https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/job/PR-958/49/ but could not find any specific report for PACT.
Please let me know if there is some gap in my understanding.

**Review by nitishsingla1234 — CHANGES_REQUESTED** [review]
Need to validate the PACT test report in Jenkins build. Please see my other comment on the PR.

**Review by shreedhar-hmcts — DISMISSED** [review]
Thanks Ruban for confirming why tests are not there. for internal professional apis. WE are good to merge this.

**Comment by ruban72** [comment]
> Need to validate the PACT test report in Jenkins build. Please see my other comment on the PR.

https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/view/change-requests/job/PR-958/48/artifact/build/reports/checkstyle/contractTest.html

**Comment by ruban72** [comment]
> Hi @ruban72 Will we be able to find the Pact test report in Jenkins build? I tried looking at https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/job/PR-958/49/ but could not find any specific report for PACT.
> Please let me know if there is some gap in my understanding.

Please see below:
https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/view/change-requests/job/PR-958/48/artifact/build/reports/checkstyle/contractTest.html

**Comment by nitishsingla1234** [comment]
Hi Ruban, the report shows only static code checkstyle issues. It does not contains the actual PACT testing result.
https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/view/change-requests/job/PR-958/48/artifact/build/reports/checkstyle/contractTest.html

**Review by nitishsingla1234 — DISMISSED** [review]

**Review by nitishsingla1234 — APPROVED** [review]

**Comment by shreedhar-hmcts** [comment]
> Hi Ruban, the report shows only static code checkstyle issues. It does not contains the actual PACT testing result.
> https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-professional-api/view/change-requests/job/PR-958/48/artifact/build/reports/checkstyle/contractTest.html

Talked with @ruban72 , he is following up with Satya to bring pact report on Jenkins. Once it is verified PR can be approved and merged.

**Review by shreedhar-hmcts — APPROVED** [review]
All looks ok now along with reporting.

**Review by nitishsingla1234 — APPROVED** [review]

**Review by abhijitdiwan — APPROVED** [review]

**Pull request merged**